pkg/input/input.go

```diff
@@ -39,3 +40,31 @@ func Password(promptText string) (string, error) {
 	fmt.Print("\n")
 	return string(bytePassword), nil
 }
+
+// Structure to hold content required for displayig prompts required for promptui library functions
+type PromptContent struct {
+	Label string
+}
+
+type PromptRunner interface {
+	Run() (int, string, error)
+}
+
+func GetYesNoSelector(pc PromptContent) *promptui.Select {
+	return &promptui.Select{
+		Label: pc.Label,
+		Items: []string{"y", "n"},
+	}
+}
+
+// Gets a y/n confirmation from the user for the given prompt content using the promptui library and returns a boolean accordingly
+func PromptGetConfirmation(runner PromptRunner) (bool, error) {
+	_, result, err := runner.Run()
+	if err != nil {
+		return false, err
+	}
+	if result == "y" {
```

Comment:
Does this have to be case sensitive? Also, should there be a default response if user doesnt enter anything. 

Since this a common package, we should allow for those options to be configured

Reply:
That's right, but we tried to be consistent with `Confirm`:
```golang
// Confirm requests a user to confirm their input
func Confirm(promptText string) (bool, error) {
	reader := bufio.NewReader(os.Stdin)
	fmt.Printf("%s (y/n) ", promptText)

	text, _ := reader.ReadString('\n')
	return strings.Trim(text, "\r\n") == "y", nil
}
```

I would say we can improve that separately sometime - and consistently for all inputs.

Reply:
We are also planning to add a non-prompt way separately for the prompt in our flow deploy command.